src/notifications/broadcaster.h

```diff
@@ -20,38 +19,40 @@
  *
  */
 
-#ifndef SRC_NOTIFICATIONS_CLIENT_H_
-#define SRC_NOTIFICATIONS_CLIENT_H_
+#ifndef SRC_NOTIFICATIONS_BROADCASTER_H_
+#define SRC_NOTIFICATIONS_BROADCASTER_H_
 #include <string>
 
 #include "notify.h"
 
 namespace Notify {
 /**
- * @brief A client which sends notifications about Posture to a server
+ * @brief A broadcaster to send messages to
+ * `Notify::NotifyReceiver`
  *
  */
-class NotifyClient {
+class NotifyBroadcast {
  private:
-  int client_fd;               ///< Socket which the client is bound to
-  struct sockaddr_in address;  ///< Client address structure
-  char* server_ip;             ///< IP address of server to connect client too
+  int broadcast_fd;            // Socket which the broadcaster is bound to
```

Comment:
I think these inline comments might need to be `///<` for Doxygen to pick them up?